HeterogeneousCore/AlpakaInterface/interface/prefixScan.h

```diff
@@ -94,9 +104,10 @@ namespace cms::alpakatools {
       ALPAKA_ASSERT_OFFLOAD(0 == blockDimension % warpSize);
       auto first = blockThreadIdx;
       auto laneId = blockThreadIdx & (warpSize - 1);
+      auto warpUpRoundedSize = (((size - 1) / warpSize) + 1) * warpSize;
```

Comment:
```suggestion
      auto warpUpRoundedSize = (size + warpSize - 1) / warpSize * warpSize;
```
to avoid issues with the corner case `size == 0`